### PR TITLE
omlibdbi: rewrite documentation

### PR DESCRIPTION
**Pull request by PascalWithopf**
Module documentation was rewritten in the new style.

### CODE REVIEW

File: source/configuration/modules/omlibdbi.rst [hunk 1]
```diff
@@ -113,16 +211,22 @@ it received limited cross-platform tests. If you run into troubles, be
...
 
 The following sample writes all syslog messages to the database
 "syslog\_db" on mysqlserver.example.com. The server is MySQL and being
 accessed under the account of "user" with password "pwd" (if you have
 empty passwords, just remove the $ActionLibdbiPassword line).
 
-::
+.. code-block:: none
+
+   $ModLoad omlibdbi $ActionLibdbiDriver mysql $ActionLibdbiHost
```
Comment (deoren) [inline source/configuration/modules/omlibdbi.rst]: Suggestion: If the [`advanced`](http://www.rsyslog.com/doc/v8-stable/configuration/conf_formats.html) configuration format is supported, convert the example to use that format.

**Comment by PascalWithopf** [comment]
@deoren I applied the suggested changes.

**Comment by deoren** [comment]
> @PascalWithopf: @deoren I applied the suggested changes.

Thanks. 

Some feedback regarding the linkage:

Instead of linking to the html version of the file, it may be better to use [Sphinx's `:doc:` cross-referencing role](http://www.sphinx-doc.org/en/stable/markup/inline.html) to link to the module docs. I've not yet attempted to covert the current docs to PDF to test the results, but when converted to epub the new links result in in references to invalid files. Using the `:doc:` role the links to the module docs work in both places.

## Suggested changes

### If you keep the explicit link text

```diff
diff --git a/source/configuration/modules/omlibdbi.rst b/source/configuration/modules/omlibdbi.rst
index c201c12..fd3d71b 100644
--- a/source/configuration/modules/omlibdbi.rst
+++ b/source/configuration/modules/omlibdbi.rst
@@ -116,10 +116,10 @@ Driver

 Name of the dbidriver to use, see libdbi-drivers documentation. As a
 quick excerpt, at least those were available at the time of this
-writiting "mysql" (suggest to use `ommysql <ommysql.html>`_ instead), "firebird"
+writiting "mysql" (suggest to use :doc:`ommysql <ommysql>` instead), "firebird"
 (Firbird and InterBase), "ingres", "msql", "Oracle", "sqlite",
 "sqlite3", "freetds" (for Microsoft SQL and Sybase) and "pgsql"
-(suggest to use `ompgsql <ompgsql.html>`_ instead).
+(suggest to use :doc:`ompgsql <ompgsql>` instead).


 Server
```

### If you allow the link caption to be automatically generated

*from the title of the page being linked to*

```diff
diff --git a/source/configuration/modules/omlibdbi.rst b/source/configuration/modules/omlibdbi.rst
index c201c12..fd3d71b 100644
--- a/source/configuration/modules/omlibdbi.rst
+++ b/source/configuration/modules/omlibdbi.rst
@@ -116,10 +116,10 @@ Driver

 Name of the dbidriver to use, see libdbi-drivers documentation. As a
 quick excerpt, at least those were available at the time of this
-writiting "mysql" (suggest to use `ommysql <ommysql.html>`_ instead), "firebird"
+writiting "mysql" (suggest to use :doc:`ommysql` instead), "firebird"
 (Firbird and InterBase), "ingres", "msql", "Oracle", "sqlite",
 "sqlite3", "freetds" (for Microsoft SQL and Sybase) and "pgsql"
-(suggest to use `ompgsql <ompgsql.html>`_ instead).
+(suggest to use :doc:`ompgsql` instead).


 Server
```

I've offered to others, but if you like I can commit back to your branch on PRs where I have feedback, or, I can leave feedback like I've been doing if you'd rather make the changes yourself. Just wanted to extend the offer.

**Comment by PascalWithopf** [comment]
@deoren ok, I will use `:doc:` in the future. And it would be great if you could commit your changes.

**Comment by deoren** [comment]
@PascalWithopf Tweaks applied and pushed to your branch. What do you think?

**Comment by PascalWithopf** [comment]
@deoren Thanks, the list is way easier to read now.

**Comment by deoren** [comment]
> @PascalWithopf: @deoren Thanks, the list is way easier to read now.

You're welcome, I'm glad you like the changes. I'll go ahead and merge this based on your feedback.

**Pull request merged**